docs/core/deploying/index.md

```diff
@@ -1,95 +1,104 @@
+- **Platform-specific targeting**\
+Because you have to publish your app for each platform, you know where your app will run. If .NET Core introduces a new platform, users can't run your app on that platform until you release a version targeting that platform. You can test your app for compatibility problems before your users run your app on the new platform.
 
-### Why deploy a framework-dependent executable?
+### Disadvantages
 
-Deploying an FDE has a number of advantages:
+- **Larger deployments**\
+Because your app includes the .NET Core runtime and all of your app dependencies, the hard drive space required is greater than a [runtime-dependent](#runtime-dependent) version.
 
-- The size of your deployment package is small. You only deploy your app and its dependencies, not .NET Core itself.
+  > [!TIP]
+  > You can reduce the size of your deployment on Linux systems by approximately 28 MB by using .NET Core [*globalization invariant mode*](https://github.com/dotnet/runtime/blob/master/docs/design/features/globalization-invariant-mode.md). This forces your app to treat all cultures like the [invariant culture](xref:System.Globalization.CultureInfo.InvariantCulture?displayProperty=nameWithType).
 
-- Multiple apps use the same .NET Core installation, which reduces both disk space and memory usage on host systems.
-
-- Your app can be run by calling the published executable without invoking the `dotnet` utility directly.
-
-There are also a few disadvantages:
-
-- Your app can run only if the version of .NET Core your app targets, [or a later version](../versions/selection.md#framework-dependent-apps-roll-forward), is already installed on the host system.
-
-- It's possible for the .NET Core runtime and libraries to change without your knowledge in future releases. In rare cases, this may change the behavior of your app.
-
-- You must publish your app for each target platform.
-
-## Step-by-step examples
-
-For step-by-step examples of deploying .NET Core apps with the .NET Core CLI, see [Publish .NET Core apps with the .NET Core CLI](deploy-with-cli.md). For step-by-step examples of deploying .NET Core apps with Visual Studio, see [Deploying .NET Core Apps with Visual Studio](deploy-with-vs.md). 
+- **Control .NET Core version**\
+.NET Core (distributed with your app) can only be upgraded by releasing a new version of your app.
 
 ## See also
 
-- [Publish .NET Core apps with the .NET Core CLI](deploy-with-cli.md)
-- [Deploying .NET Core Apps with Visual Studio](deploy-with-vs.md)
-- [Packages, Metapackages and Frameworks](../packages.md)
-- [.NET Core Runtime IDentifier (RID) catalog](../rid-catalog.md)
+- [Deploying .NET Core Apps with .NET Core CLI.](deploy-with-cli.md)
+- [Deploying .NET Core Apps with Visual Studio.](deploy-with-vs.md)
+- [Packages, Metapackages, and Frameworks.](../packages.md)
```

Comment:
@dsplaisted Can you check this link https://docs.microsoft.com/en-us/dotnet/core/packages. It appears out of date. If so, let's remove it here.

Reply:
Yes, that is super out of date.  I think we should delete that page entirely, as it doesn't look like any of it applies any more. @mairaw @KathleenDollard

Reply:
@dsplaisted None of it applies anymore? What about for 2.1? I'll remove the link but I think we should open another issue to evaluate this.